java/flinktestmock/src/main/java/org/apache/flink/core/fs/FileStatus.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This file is based on source code from the Hadoop Project (http://hadoop.apache.org/), licensed
+ * by the Apache Software Foundation (ASF) under the Apache License, Version 2.0. See the NOTICE
+ * file distributed with this work for additional information regarding copyright ownership.
+ */
+
+package org.apache.flink.core.fs;
```

Comment:
Will this result in potential package conflicts?

Reply:
No. These files are placed in a separate package and are not included in javalib's SOURCES file[1], so they will not be compiled into Forst's javalib.

[1] https://github.com/ververica/ForSt/blob/ec88681c32e5f9d80b0bf331070bd05d740d685c/java/Makefile#L293C1-L293C56